src/styles/base.css

```
@@ -135,3 +135,7 @@ dialog {
 p {
   text-wrap: pretty;
 }
+
+starlight-menu-button > button > svg {
```

Ah sorry, we also need to select for dark mode
```suggestion
:root[data-theme="dark"] starlight-menu-button > button > svg {
```